src/lib/post.ts

```diff
@@ -25,7 +26,7 @@ const getAll = async (req: IGetUserAuthRequest, res: Response) => {
 			posts = await Post.find({})
 				.skip(((page as number) - 1) * (limit as number))
 				.limit(limit as number)
-				.populate('createdBy', '-password -tokens')
+				.populate('createdBy', 'name email')
 				.sort({ createdAt: -1 })
 				.where('deleted')
 				.equals(false);
```

Comment:
please add filters as json object in find() params

Reply:
Done